tests/e2e/run-e2e-test.sh

```diff
@@ -52,10 +52,10 @@ function getArrayInfo() {
 
 function vaultSetupAutomation() {
   echo "Removing any existing vault installation..."
-  helm delete vault || true
+  helm delete vault0 || true
   echo "Installing vault with all secrets for Authorization tests..."
   cd ./scripts/vault-automation
-  go run main.go --kubeconfig ~/.kube/config --name vault --env-config
+  go run main.go --kubeconfig ~/.kube/config --name vault0 --env-config --secrets-store-csi-driver true
```

Comment:
Update to `--secrets-store-csi-driver=true` otherwise any following flags will be ignored